bundle/artifacts/whl/build.go

```diff
@@ -51,11 +52,15 @@ func (m *build) Apply(ctx context.Context, b *bundle.Bundle) error {
 	}
 	cmdio.LogString(ctx, fmt.Sprintf("artifacts.whl.Build(%s): Build succeeded", m.name))
 
-	wheel := python.FindFileWithSuffixInPath(distPath, ".whl")
-	if wheel == "" {
+	wheels := python.FindFilesWithSuffixInPath(distPath, ".whl")
+	if len(wheels) == 0 {
 		return fmt.Errorf("artifacts.whl.Build(%s): cannot find built wheel in %s", m.name, dir)
 	}
-	artifact.File = filepath.Join(dir, wheel)
+	for _, wheel := range wheels {
```

Comment:
When can we expect multiple wheels?

Reply:
For Python project I can imagine we might have some setup for monorepo and custom builds which yields multiple Python wheels.

Reply:
In that case I'd expect multiple artifacts be defined, one for each wheel. But it's fine for symmetry with the Java case you describe below.